README.md

```diff
@@ -14,6 +14,10 @@ git@github.com:AndelaOSP/Andela-Socials.git
 
 Create a `.env` file with the content of the `.env.sample` and edit with your personal details.
 
+Check that the node version you are running is version 8. Otherwise you can use nvm to help manage the node version you are running.
+Follow this [guide](https://gist.github.com/d2s/372b5943bce17b964a79#install-nvm-for-managing-nodejs-versions) to help install nvm.
+You would then run `nvm install v8.0` to install node version 8 and `nvm use v8` to use that specific node version, but this will only work with that running bash session. To make version 8 the default node version for your system you would use `nvm alias default 8.0` 
```

Comment:
Welldone on the PR. I think for consistency across the board, especially for some of us using nvm, I would like to suggest that a `.nvmrc` with `8` as a value be added to the code base. With that, anyone can easily run `nvm use` without worrying about version the codebase uses.

Also, like I suggested in one of my comments on slack, some of us already use a version of node higher than 8 in existing projects. Suggesting that version 8 be made default will break a lot of things in those projects. For example, I can use `async await` in node 10 but can't use that in node 8.

Reply:
@malmike rather than suggesting that all users should use a node version 8, can you rather suggest the minimum node version needed to effectively run the project.

Can you enlighten me on the exact error you got when you tried to run the project on a node version different from version 8?

Reply:
Can you enlighten me on the exact error you got when you tried to run the project on a node version different from version 8?

I'm glad this came up now, we'd need to define a node version standard for this project

Reply:
There is a library called node-gyp@3.6.2 that asks for python2 instead of python3 which leads to a configuration error.
It also points to an error with node-sass. On switching to node v8, it runs smoothly

Reply:
I actually don't have up to node v8 and this project still worked for me. Is this really needed?